### ISSUE TITLE
Build Error: Editor namespace missing

### ISSUE THREAD

**Issue by MichaelTGood** [Kurloc/ColoredBoxGroup#3]
Hey man,

I found out today that I haven't built a version of my project since implementing your plugin, because it immediately gave me many build errors when doing so, all pointing at the Drawer script, and pretty much any line that points to anything labelled "Editor". 

After some testing with different folders and Asmdefs, I realized the simple answer was to add `#if UNITY_EDITOR` at the top (and of course `#endif` at the bottom) of `ColoredBoxGroupDrawer`.

Is there another way to deal with the problem? Has it not come up before, or am I the first person to build a project with the plugin? 😅

**Comment by Kurloc** [comment]
I actually have come across a problem similar to this when building a
project personally, this was 3 months ago at least, but I recall having to
stuff #if UNITY_EDITOR in quite a few places. It's definitely worth a quick
push/fix!

**Comment by Kurloc** [comment]
It's been so long, but I thought there was perhaps another issue/fix, I had a lot of custom odin stuff in my project where I had to deal with this issue on a few of them. I did some digging though.

I was thinking maybe they used a special attribute for this to prefix their odin attributes, but it looks like it's already on the ColoredBoxGroup.cs as `[System.Diagnostics.Conditional("UNITY_EDITOR")]`. I think this combined with #if UNITY_EDITOR is going to be the fix used/seen in other custom Odin attributes.
